packages/zoe/src/contractSupport/ratio.js

```diff
@@ -0,0 +1,135 @@
+import './types';
```

Comment:
```suggestion
// @ts-check
import './types';
```

We need to make sure to add the type checking in more places, but any files that we add, let's make sure to add this.

Reply:
done